extensions/bokeh_magic.py

```diff
@@ -27,6 +31,11 @@
 class BokehMagics(Magics):
     """Magic to embed Bokeh into the IPython notebook."""
 
+    if old_bokeh:
+        raise RuntimeError("We are deprecating the Bokeh magic. "
```

Comment:
Showing it at all by default is more chatty than standard python, which does not show deprecation warnings at all unless specifically requested. 

> On Feb 12, 2015, at 11:58 AM, Damián Avila <notifications@github.com> wrote:
> 
> In extensions/bokeh_magic.py:
> 
> > @@ -27,6 +31,11 @@
> >  class BokehMagics(Magics):
> >      """Magic to embed Bokeh into the IPython notebook."""
> >  
> > +    if old_bokeh:
> > +        raise RuntimeError("We are deprecating the Bokeh magic. "
> 
> It is no more chatty than with the RuntimeError but I agree DeprecationWarning is the proper exception to raise... I already pushed the change.
> 
> —
> Reply to this email directly or view it on GitHub.
>